pkg/plugin/sdk/deployment_test.go

```diff
@@ -658,3 +658,173 @@ func TestDetermineVersionsResponse_toModel(t *testing.T) {
 		})
 	}
 }
+
+func TestNewDetermineStrategyRequest(t *testing.T) {
```

Comment:
[nit] We can add `t.Parallel()`

Reply:
Thank you.
Fixed on https://github.com/pipe-cd/pipecd/pull/5655/commits/6f6ee13b8daaf4c42724fe021986cb097a624bec